StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+        {
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("guestLogin")).Displayed);
+            IWebElement GuestLogin = Driver.FindElement(By.ClassName("guestLogin"));
+            GuestLogin.FindElement(By.Id("checkoutContinueGuestWebsphere")).Click();
+        }
+
+        [When(@"Nearest UPS point is selected")]
+        public void WhenNearestUPSPointIsSelected()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("checkoutColumn")).Displayed);
+            IWebElement RadioButton1 = Driver.FindElement(By.ClassName("deliveryAddress"));
+            var UPSRadioButton = RadioButton1.FindElements(By.ClassName("methodName"));
+            UPSRadioButton.ElementAt(0).Click();
+            Wait.Until(Driver => Driver.FindElement(By.Id("pupContainer")).Displayed);
+            Driver.SwitchTo().Frame("pupContainer");
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("iframeDialog")).Displayed);
+            IWebElement bewerken = Driver.FindElement(By.Id("editLink"));
+            bewerken.FindElement(By.Name("EditLink")).Click();
+            IWebElement InvulVelden = Driver.FindElement(By.ClassName("iframeDialog"));
+            InvulVelden.FindElement(By.Name("AddressLine1")).SendKeys("Danzigerkade 165");
+            InvulVelden.FindElement(By.Name("City")).SendKeys("Amsterdam");
+            InvulVelden.FindElement(By.Name("PostalCode")).SendKeys("1013 AP");
+            InvulVelden.FindElement(By.Name("FindButton")).Click();
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("subcol5")).Displayed);
+            IWebElement LocationSelector = Driver.FindElements(By.ClassName("subcol5")).ElementAt(0);
+            LocationSelector.FindElement(By.Name("SelectLocation")).Click();
+            Driver.SwitchTo().DefaultContent();
+            System.Threading.Thread.Sleep(1000);
+        }
+         
+        [When(@"I proceed to the payment page")]
+        public void WhenIProceedToThePaymentPage()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.Id("pickUpPointForm")).Displayed);
+            ((IJavaScriptExecutor)Driver).ExecuteScript("arguments[0].scrollIntoView(true);", Driver.FindElement(By.Id("pickUpPointForm")));
+            IWebElement persDetails = Driver.FindElement(By.Id("pickUpPointForm"));
+            IWebElement TitleSelector = persDetails.FindElement(By.Id("pickUpPoint_title_chosen"));
+            TitleSelector.Click();
+            IWebElement TitleSelected = TitleSelector.FindElement(By.ClassName("chosen-drop"));
+            TitleSelected.FindElements(By.ClassName("active-result")).ElementAt(1).Click();
+            //var dropdownpers = Driver.FindElements(By.ClassName("active-result")).Where(x => x.Displayed);
+            //dropdownpers.ElementAt(1).Click();
+            IWebElement Voornaam = Driver.FindElement(By.ClassName("pupFirstName"));
+            Voornaam.FindElement(By.Name("firstName")).SendKeys("Theo");
```

Comment:
what is Voornaam... although I know it, but please use english, otherwise I will write Chinese review here :)

Reply:
and all below....